pydm/widgets/image.py

```diff
@@ -287,18 +287,39 @@ def image_width_changed(self, new_width):
             return
         self._image_width = int(new_width)
 
+    @staticmethod
+    def process_image(image):
+        """
+        Boilerplate method to be used by applications in order to add calculations and also modify the image before
```

Comment:
Docstring should be one line 😄

Reply:
I'm not sure, but I think, by pep8, docstrings should have a short summary line (< 79 characters, ending with a period), followed by a blank line and then a description.